views/templates/hook/envoimoinscher_carrier.tpl

```diff
@@ -76,21 +76,91 @@ var carrier_translation = {
 <script type="text/javascript" src="{$baseDir|escape:'htmlall'}modules/envoimoinscher/js/carrier.js"></script>
 <link type="text/css" rel="stylesheet" href="{$baseDir|escape:'htmlall'}modules/envoimoinscher/css/carrier.css" />
 <script type="text/javascript"> 
-  if(typeof idAddress == "undefined") {
+
+// create variables
+
+
+// handle google map loading
+function loadScript() {
+  var script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.src = '//maps.googleapis.com/maps/api/js?v=3.exp' +
+      '&signed_in=true&callback=initialize_gmap';

```

Comment:
In some templates addJs function doesn't work and that's
because we use this function to prevents those cases.